content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -129,10 +136,163 @@ the previous provider.
+
+  .. image:: electronic_invoicing/peppol-system-parameter.png
+     :alt: peppol test mode
+
+Now you can request a verification code to be sent to you
+by clicking :guilabel:`Verify phone number`.
+
+.. image:: electronic_invoicing/peppol-registration-verify.png
+   :alt: phone validation request verification
+
+A text message containing a code is sent to the phone number provided to finalize
+the registration process.
 
 .. image:: electronic_invoicing/phone-registration.png
    :alt: phone validation
 
-All invoices and vendor bills are now sent directly using the Peppol network.
+Once you enter the code and click :guilabel:`Confirm`, you will see that your
+registration is pending activation.
+
+.. image:: electronic_invoicing/peppol-registration-pending.png
+   :alt: pending application
+
+It should be automatically activated within a day.
+In the meantime, you can set up the default journal here to receive incoming
+vendor bills.
+
+.. tip::
+  It is also possible to manually trigger the cron that checks the registration status,
+  by going to :menuselection:`Settings --> Technical --> Scheduled Actions --> PEPPOL\: update participant status`
+
+Your application status is going to be updated soon after you are
+registered on the Peppol network.
+
+.. image:: electronic_invoicing/peppol-registration-active.png
+   :alt: active application
+
+All invoices and vendor bills can now be sent directly using the Peppol network.
+
+Partner verification
+--------------------
+
+Before sending an invoice to a partner using the Peppol network, you need to verify
+that they are indeed registered as a Peppol participant as well.
+
+Go to :menuselection:`Accounting --> Customers --> Customers` and open the customer's form.
+Then go to :menuselection:`Accounting tab --> Electronic Invoicing`, select the correct format,
+and make sure their `Peppol EAS` code and the `Endpoint` are filled in.
+Then, click :guilabel:`Verify`. If the partner exists on the network, their `Peppol endpoint validity`
+will be set as Valid.
+
+.. image:: electronic_invoicing/peppol-partner-verify.png
+   :alt: verify partner registration
+
+.. important::
+   While Odoo prefills both the EAS code and the Endpoint number based on the information
+   available for a partner, it is better to confirm directly with the partner,
+   as these are essentially their contact details on the Peppol network.
+
+Send invoices
+-------------
+
+When you are ready to send an invoice via Peppol network, simply click :guilabel:`Send & Print`
+on the invoice form or select multiple invoices in the list view and go to
+:menuselection:`Actions --> Send & Print`.
+
+Make sure that both `BIS Billing 3.0` and `Send via PEPPOL` checkboxes are ticked.
+
+.. image:: electronic_invoicing/peppol-send-print.png
+   :alt: send peppol invoice
+
+The invoices will be queued to be sent in a batch if you send several of them at once.
+Once they are sent via Peppol, the status will be changed to `Processing`.
```

Comment:
The invoices get queued to be sent in a batch if you send several simultaneously. Once they are sent via Peppol, the status will be changed to :guilabel:`Processing`.